pyproject.toml

```diff
@@ -9,53 +9,52 @@ license = "MIT"
 python = "3.10.*"
 
 # See https://bot-core.pythondiscord.com/ for docs.
-bot-core = { url = "https://github.com/python-discord/bot-core/archive/refs/tags/v8.2.1.zip", extras = ["async-rediscache"] }
-redis = "4.3.4"
-fakeredis = { version = "1.9.3", extras = ["lua"] }
+pydis_core = { version = "9.1.1", extras = ["async-rediscache"] }
+redis = "4.3.5"
+fakeredis = { version = "2.0.0", extras = ["lua"] }
 
 aiohttp = "3.8.3"
 arrow = "1.2.3"
 beautifulsoup4 = "4.11.1"
-colorama = { version = "0.4.5", markers = "sys_platform == 'win32'" }
+colorama = { version = "0.4.6", markers = "sys_platform == 'win32'" }
 coloredlogs = "15.0.1"
-deepdiff = "5.8.1"
-emoji = "2.1.0"
+deepdiff = "6.2.1"
+emoji = "2.2.0"
 feedparser = "6.0.10"
 lxml = "4.9.1"
 
 # Must be kept on this version unless doc command output is fixed
 # See https://github.com/python-discord/bot/pull/2156
 markdownify = "0.6.1"
 
-more-itertools = "8.14.0"
+more-itertools = "9.0.0"
```

Comment:
Breaking changes are only to functions we do not use.
https://more-itertools.readthedocs.io/en/stable/versions.html